tests/spread/plugins/v2/build-and-run-hello/task.yaml

```diff
@@ -43,6 +45,8 @@ execute: |
   cd "../snaps/$SNAP"
 
   # Build what we have and verify the snap runs as expected.
+  snapcraft prime
```

Comment:
The only suggestion I would have here is to do the snap install and verify both times.  Just to make sure the first build works ;)